configure.ac

```diff
@@ -1210,6 +1210,46 @@ if test "$enable_izem_queue" != "no" && test "$enable_izem_queue" != "none"; the
     fi
 fi
 
+# YAKSA
+AC_ARG_VAR([YAKSALIBNAME],[can be used to override the name of the YAKSA library (default: "yaksa")])
+YAKSALIBNAME=${YAKSALIBNAME:-"yaksa"}
+export YAKSALIBNAME
+AC_SUBST(YAKSALIBNAME)
+AC_ARG_WITH([yaksa-prefix],
+            [AS_HELP_STRING([[--with-yaksa-prefix[=DIR]]],
+                            [use the YAKSA library installed in DIR,
+                             rather than the one included in src/yaksa.  Pass
+                             "embedded" to force usage of the YAKSA source
+                             distributed with MPICH.])],
+            [],dnl action-if-given
+            [with_yaksa_prefix=embedded]) dnl action-if-not-given
+
+yaksasrcdir=""
+AC_SUBST([yaksasrcdir])
+yaksalibdir=""
+AC_SUBST([yaksalibdir])
+yaksalib=""
+AC_SUBST([yaksalib])
+
+if test "$with_yaksa_prefix" = "embedded" ; then
+    # no need for libtool versioning when embedding YAKSA
+    yaksa_subdir_args="--disable-versioning --enable-embedded"
```

Comment:
We might need to modify yaksa to accept the `--enable-embedding` flag (which can also disable versioning).  Right now it doesn't do that.

Reply:
The `--disable-versioning` flag needs to be removed.  Yaksa would do that when embedding is enabled.